core/pom.xml

```diff
@@ -221,9 +213,10 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.springframework</groupId>
-      <artifactId>spring-context</artifactId>
-      <version>${dependency.org.springframework.spring-context.version}</version>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-platform-core</artifactId>
+      <version>${platform.version}</version>
+      <classifier>tests</classifier>
```

Comment:
This dependency is duplicated but with different scopes: https://github.com/pentaho/pentaho-metaverse/pull/580/files#diff-357e4854869b2e21c38b1b437f11095aR103

Only use one declaration.

Reply:
Agree, I'll fix it